packages/dbs-ai/server/lib/Smarti.js

```diff
@@ -160,6 +160,33 @@ Meteor.methods({
 		SystemLogger.debug('Smarti - last smarti result requested:', JSON.stringify(rid, '', 2));
 		SystemLogger.debug('Smarti - last message:', JSON.stringify(RocketChat.models.LivechatExternalMessage.findOneById(rid), '', 2));
 		return RocketChat.models.LivechatExternalMessage.findOneById(rid);
+	},
+
+	/**
+	 * A function searching conversations with the given text in the given rooms
+	 *
+	 * @param {string} text - the text we should search for
+	 * @param {[]} accessibleRooms - an array containing all rooms in which given text should be searched
+	 * @param {[]} filter - any additional filters; not used right now
+	 * @param {number} start - pagination offset should be a multiple of rows
+	 * @param {number} rows - number of results being returned
+	 */
+	searchConversations(text, accessibleRooms, filter = null, start = 0, rows = 10) {
```

Comment:
@lobo87 a less generic name might help - it's the Smarti-Search, so `findInSmarti` would be one suggestion

Reply:
ill change that

Reply:
@mrsimpson meteor function is now called "findInSmarti" as suggested. The function in the Smarti adapter is called onSearch to conform to the other functions in there.